package.json

```
@@ -0,0 +1,61 @@
+{
+	"name": "cli",
+	"version": "0.0.0",
+	"license": "MIT",
+	"bin": {
+		"asyncapi": "dist/cli.js",
+		"@asyncapi/cli": "dist/cli.js"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"scripts": {
+		"build": "tsc",
+		"start": "npm run build && node dist/cli.js",
+		"pretest": "npm run build",
+		"pretest:coverage": "npm run build",
+		"test": "jest",
+		"test:coverage": "jest --verbose --coverage"
+	},
+	"files": [
+		"dist/cli.js"
+	],
+	"dependencies": {
+		"@asyncapi/parser": "^1.4.5",
+		"commander": "^7.1.0",
+		"ink": "^3.0.8",
+		"meow": "^9.0.0",
+		"react": "^17.0.1",
+		"reflect-metadata": "^0.1.13",
+		"tsyringe": "^4.5.0"
+	},
+	"devDependencies": {
+		"@sindresorhus/tsconfig": "^0.9.0",
```

please add those to enable release automation so once we merge this PR, we get it published to npm

```suggestion
                "@semantic-release/commit-analyzer": "^8.0.1",
                "@semantic-release/github": "^7.2.3",
                "@semantic-release/npm": "^7.1.3",
                "@semantic-release/release-notes-generator": "^9.0.2",
                "conventional-changelog-conventionalcommits": "^4.4.0",
                "semantic-release": "^17.4.3",
		  "@sindresorhus/tsconfig": "^0.9.0",
```

---

after adding them, please update package-lock

---

@jotamusik don't forget to add these dependencies

---

done!